packages/forms/frontend/sirius-components-forms/src/propertysections/TreePropertySection.tsx

```diff
@@ -142,14 +149,18 @@ const TreeItem = ({ node, nodes, readOnly, editingContextId, formId, widgetId }:
 
   const handleClick: React.MouseEventHandler<HTMLDivElement> = () => {
     if (node.selectable) {
+      const treePath = node.id.split('/');
+      const nodeId = treePath[treePath.length - 1];
       const newSelection: SelectionEntry = {
-        id: node.id,
+        id: nodeId,
```

Comment:
Why does this need to change? I don't think that you are changing the property of the node object in this PR unless I'm missing something?